FloripaBus.Tests/Repositories/RoutesRepositoryTests.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using NUnit.Framework;
+using System.Net.Http;
+using System.Net;
+using System.Text;
+
+namespace FloripaBus.Tests
+{
+	[TestFixture]
+	public class RoutesRepositoryTests

```

Comment:
Code coverage! Thanks for creating unit tests. Although, all the tests in this test class did not run succesfullly.

Below is the outcome I have:

![screen shot 2015-09-29 at 2 29 09 pm](https://cloud.githubusercontent.com/assets/11648468/10172616/adf895c4-66b6-11e5-8067-37e71598b129.png)

Do you know what I have to do to make them run?